qconsolidate.py

```diff
@@ -24,20 +26,30 @@
 # to the Free Software Foundation, 51 Franklin Street, Suite 500 Boston,
 # MA 02110-1335 USA.
 #
-#******************************************************************************
+# *****************************************************************************
+
 
-import qgis
+import qgis  # NOQA
 
-from PyQt4.QtCore import *
-from PyQt4.QtGui import *
+from qgis.PyQt.QtCore import (
+                              QCoreApplication,
+                              QFileInfo,
+                              QLocale,
+                              QSettings,
+                              QTranslator,
+                              )
+from qgis.PyQt.QtGui import (
+                             QMessageBox,
+                             QAction,
+                             QIcon,
+                             )
 
-from qgis.core import *
-from qgis.gui import *
+from qgis.core import QGis, QgsApplication
 
 import qconsolidatedialog
 import aboutdialog
 
-import resources_rc
+import resources_rc  # NOQA
```

Comment:
Still required?

Reply:
I am almost sure it's required only here, and I've removed it where I believe it was not needed.